module/zcommon/zfs_fletcher_sse.c

```diff
@@ -48,7 +48,7 @@
 #include <zfs_fletcher.h>
 
 struct zfs_fletcher_sse_array {
-	uint64_t v[2] __attribute__((aligned(16)));
+	uint64_t v[2];
```

Comment:
I would leave the alignment as before. Some CPUs will run unaligned movs as fast as aligned when the address is properly aligned.